modular-docs-manual/content/topics/module_creating-assemblies.adoc

```diff
@@ -0,0 +1,32 @@
+[[creating-assemblies]]
+= Creating Assemblies
+
+An assembly is a representation of a user story, so you need to create an assembly for each user story that you defined.
+
+An assembly could represent an article, "chapter" in a book, or even an entire book. However, one of the benefits of modular documentation is that you do not need to worry about how the assembly will ultimately be used -- each assembly represents a user goal, and once you create it, it can be "included" anywhere it is needed (a publication, within another assembly, and so on).
+
+.Procedure
+
+. Create an assembly file for each user story that you identified.
++
+--
+Be sure to follow a consistent file naming standard. For example, you could use the following file naming pattern:
+
+`assembly_configuring-security.adoc`
+--
+
+. For each top-level assembly file that you created, fill in the content:
```

Comment:
We're already explaining what to include in an assembly in the template and in the section for writing guidelines. Do we want to repeat it here? For the part about creating modules, you just linked to the guidelines, and I think that was perfectly okay.

Reply:
Done.